R/est.incidence.R

```diff
@@ -51,13 +55,15 @@ est.incidence <- function(
   }
 
   .errorCheck(
-    data = pop_data,
+    check_data = pop_data,
     antigen_isos = antigen_isos,
     curve_params = curve_params)
 
   pop_data = pop_data %>%
     dplyr::filter(.data$antigen_iso %in% antigen_isos) %>%
-    dplyr::select("value", "age", "antigen_iso") %>%
+    dplyr::select(.data$value,
+                  .data$age,
+                  "antigen_iso") %>%
```

Comment:
what happens here when `standardize = FALSE` in `load_pop_data()`? I suspect it will break. 

Please write unit tests for `est.incidence()` under both scenarios (`standardize = FALSE` and `standardize = TRUE`)